unicorecmsunfpa/templates/home.pt

```diff
@@ -13,7 +13,7 @@
                     <condition tal:omit-tag="" tal:condition="not (localisation and localisation.image_host and localisation.image)">
```

Comment:
This is the offending block. It includes a default image. Rather just remove it here, alas you'll be requested in 2 days time to upload the "new" homepage image and you'd have to go through this again, whereas leaving the one above will allow you to set the image via the CMS using `localisation.image`